twilio/rest/accounts/v1/__init__.py

```diff
@@ -1,64 +1,60 @@
-# coding=utf-8
-r"""
-This code was generated by
-\ / _    _  _|   _  _
- | (_)\/(_)(_|\/| |(/_  v1.0.0
-      /       /
+"""
+    This code was generated by
+    ___ _ _ _ _ _    _ ____    ____ ____ _    ____ ____ _  _ ____ ____ ____ ___ __   __
+    |  | | | | |    | |  | __ |  | |__| | __ | __ |___ |\ | |___ |__/ |__|  | |  | |__/
+    |  |_|_| | |___ | |__|    |__| |  | |    |__] |___ | \| |___ |  \ |  |  | |__| |  \
+
+    Twilio - Accounts
+    This is the public Twilio REST API.
+
+    NOTE: This class is auto generated by OpenAPI Generator.
+    https://openapi-generator.tech
+    Do not edit the class manually.
 """
 
 from twilio.base.version import Version
-from twilio.rest.accounts.v1.auth_token_promotion import AuthTokenPromotionList
-from twilio.rest.accounts.v1.credential import CredentialList
-from twilio.rest.accounts.v1.secondary_auth_token import SecondaryAuthTokenList
+from twilio.base.domain import Domain
+from twilio.rest.accounts.v1.auth_token_promotion import AuthTokenPromotionListInstance
+from twilio.rest.accounts.v1.credential import CredentialListInstance
+from twilio.rest.accounts.v1.secondary_auth_token import SecondaryAuthTokenListInstance
```

Comment:
Should these be changing from `List` to `ListInstance`? If we can do without the change, that'd be better.

Reply:
The change happens [here](https://github.com/twilio/twilio-oai-generator/blob/main/src/main/java/com/twilio/oai/DirectoryStructureService.java#L159). It's shared code so it would require a decent bit of refactoring to make it only affect Python.

Reply:
Could use the `resourceName`: https://github.com/twilio/twilio-oai-generator/blob/475e098f916a6f844ed83b25f8805031bae5d0fa/src/main/java/com/twilio/oai/DirectoryStructureService.java#L163